inference-engine/src/preprocessing/ie_preprocess_data.cpp

```diff
@@ -779,6 +779,7 @@ class PreProcessData : public IPreProcessData {
     Blob::Ptr getRoiBlob() const override;
 
     void execute(Blob::Ptr &outBlob, const PreProcessInfo& info, bool serial, int batchSize = -1) override;
+    void execute(const Blob::Ptr& inBlob, Blob::Ptr &outBlob, const PreProcessInfo& info, bool serial, int batchSize = -1) override;
```

Comment:
I think it is better to rename `outBlob` to `preprocessedBlobs` as inference requests also has some outBlobs.

Reply:
`inBlob` to `userBlob`?

Reply:
renamed `outBlob` to `preprocessedBlob` and `_roiBlob` to  `_userBlob`. (though renaming of according methods is postponed)